File: C++/module_rect.cpp
#include <math.h>
#include <iostream>

float square(float x, float y){
	return x * y;}

float perimeter(float x, float y){
	return (x + y) * 2;}

float diagon(float x, float y){
	return sqrt(x * x + y * y);}

using namespace std;

int main(){
	float x = 3;
	float y = 4;
	cout << square(x, y) << " " << perimeter(x, y) << " " << diagon(x, y)<< endl;
}

File: GO/module_rect.go
package rect
import (
"math"
)

func square(x, y float64) float64{
	return x * y}
	
func perimeter(x, y float64) float64{
	return (x + y) * 2} \\ returning perim
	
func diagon(x, y float64) float64{
	var z float64
	z = math.Sqrt(x * x + y * y)
	return z}
